samples/dfu/nrf5340/src/main.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/device.h>
+#include <dfu_target_uart_host.h>
+#include <zephyr/logging/log.h>
+#include <dfu/dfu_target_mcuboot.h>
+LOG_MODULE_REGISTER(app, CONFIG_APP_LOG_LEVEL);
+
+/* CONFIG_DOWNLOAD_CLIENT_BUF_SIZE + some room */
+static uint8_t buf[2100];
```

Comment:
Well it's not `CONFIG_DOWNLOAD_CLIENT_BUF_SIZE` plus some room because `CONFIG_DOWNLOAD_CLIENT_BUF_SIZE` can be set to any arbitrary value, update to use the `CONFIG_DOWNLOAD_CLIENT_BUF_SIZE` value